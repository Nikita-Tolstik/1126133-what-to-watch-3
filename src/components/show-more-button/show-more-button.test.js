import React from 'react';
import renderer from 'react-test-renderer';
import ShowMoreButton from './show-more-button.jsx';

it(`Render ShowMoreButton - isShowMoreButton = true`, () => {
  const tree = renderer.create(
      <ShowMoreButton
        isShowMoreButton={true}
        onShowMoreButtonClick={() => {}}
      />
  ).toJSON();

  expect(tree).toMatchSnapshot();
});

it(`Render ShowMoreButton - isShowMoreButton = false`, () => {
  const tree = renderer.create(
      <ShowMoreButton
        isShowMoreButton={false}
        onShowMoreButtonClick={() => {}}
      />
  ).toJSON();

  expect(tree).toMatchSnapshot();
});
