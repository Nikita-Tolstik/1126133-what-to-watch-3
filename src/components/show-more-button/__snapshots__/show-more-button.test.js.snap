// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render ShowMoreButton - isShowMoreButton = false 1`] = `null`;

exports[`Render ShowMoreButton - isShowMoreButton = true 1`] = `
<div
  className="catalog__more"
>
  <button
    className="catalog__button"
    onClick={[Function]}
    type="button"
  >
    Show more
  </button>
</div>
`;
